eras/alonzo/impl/src/Cardano/Ledger/Alonzo/TxBody.hs

```diff
@@ -300,79 +300,53 @@ pattern AlonzoTxBody ::
   StrictMaybe Network ->
   AlonzoTxBody era
 pattern AlonzoTxBody
-  { inputs,
-    collateral,
-    outputs,
-    txcerts,
-    txwdrls,
-    txfee,
-    txvldt,
-    txUpdates,
-    reqSignerHashes,
-    mint,
-    scriptIntegrityHash,
-    adHash,
-    txnetworkid
+  { atbInputs,
+    atbCollateral,
+    atbOutputs,
+    atbCerts,
+    atbWdrls,
+    atbTxFee,
+    atbValidityInterval,
+    atbUpdate,
+    atbReqSignerHashes,
+    atbMint,
+    atbScriptIntegrityHash,
+    atbAuxDataHash,
+    atbTxNetworkId
   } <-
   TxBodyConstr
     ( Memo
-        TxBodyRaw
-          { _inputs = inputs,
-            _collateral = collateral,
-            _outputs = outputs,
-            _certs = txcerts,
-            _wdrls = txwdrls,
-            _txfee = txfee,
-            _vldt = txvldt,
-            _update = txUpdates,
-            _reqSignerHashes = reqSignerHashes,
-            _mint = mint,
-            _scriptIntegrityHash = scriptIntegrityHash,
-            _adHash = adHash,
-            _txnetworkid = txnetworkid
+        AlonzoTxBodyRaw
+          { atbrInputs = atbInputs,
+            atbrCollateral = atbCollateral,
+            atbrOutputs = atbOutputs,
+            atbrCerts = atbCerts,
+            atbrWdrls = atbWdrls,
+            atbrTxFee = atbTxFee,
+            atbrValidityInterval = atbValidityInterval,
+            atbrUpdate = atbUpdate,
+            atbrReqSignerHashes = atbReqSignerHashes,
+            atbrMint = atbMint,
+            atbrScriptIntegrityHash = atbScriptIntegrityHash,
+            atbrAuxDataHash = atbAuxDataHash,
+            atbrTxNetworkId = atbTxNetworkId
           }
         _
       )
   where
-    AlonzoTxBody
-      inputsX
-      collateralX
-      outputsX
-      certsX
-      wdrlsX
-      txfeeX
-      vldtX
-      updateX
-      reqSignerHashesX
-      mintX
-      scriptIntegrityHashX
-      adHashX
-      txnetworkidX =
-        mkAlonzoTxBody $
-          TxBodyRaw
-            inputsX
-            collateralX
-            outputsX
-            certsX
-            wdrlsX
-            txfeeX
-            vldtX
-            updateX
-            reqSignerHashesX
-            mintX
-            scriptIntegrityHashX
-            adHashX
-            txnetworkidX
+    AlonzoTxBody inputs collateral outputs certs wdrls txfee validityInterval update reqSignerHashes mint scriptIntegrityHash auxDataHash txnetworkid =
+      mkAlonzoTxBody $
+        AlonzoTxBodyRaw inputs collateral outputs certs wdrls txfee validityInterval update reqSignerHashes mint scriptIntegrityHash auxDataHash txnetworkid
```

Comment:
Sorry for asking you to do all this mundane stuff, but could you switch this to use record syntax, it would be much cleaner and follow the pattern above. Something along the lines:
```
AlonzoTxBody {atbInputs, ...} =
  mkAlonzoTxBodyRaw $
    AlonzoTxBodyRaw {
       atbrInputs = atbrInputs,
       ....
       }
````

Reply:
Sure! :smile: I did think about it before too, but thought that using it in one place will give enough safety. I will switch it out.